.vuepress/config.js

```diff
@@ -192,8 +194,54 @@ export default defineUserConfig({
       },
     }),
     shikiPlugin({
-      theme: 'dark-plus',
+      themes: {
+        dark: 'dark-plus',
+        onedarkpro: 'one-dark-pro', // pre-load one-dark-pro for ansi code blocks
+      },
       lineNumbers: 10,
+      transformers: [
+        transformerRenderWhitespace(),
+        {
+          // highlight nushell code blocks with nu-highlight
+          preprocess(code, options) {
+            if (this.options.lang != 'nushell' && this.options.lang != 'nu') {
+              return code;
+            }
+
+            this.options.defaultColor = 'onedarkpro';
+            // this doesn't work at the top-level for some reason
+            this.options.colorReplacements = {
+              // make one-dark-pro background color the same as dark-plus
+              '#282c34': '#1e1e1e',
+              // HACK: change color of comments, since nu-highlight can't highlight them
+              '#abb2bf': '#80858f',
+            };
```

Comment:
It's better to extract these a few lines to a function like adjustTheme, so we can reuse it later